docs/wxm-devices/D1-bundle/D1-connect-and-claim.mdx

```diff
@@ -153,22 +153,22 @@ Your first rewards will appear after 24 hours of the station being online.
 
 <hr/>
 
-### Type the WG1000 Gateway serial number when prompted
+### Prepare for claiming you WG1200Gateway
 
 <div class="explanation">
-        <img style={{width:"calc(100%*1/3 - 10px)"}} src={useBaseUrl("/img/wxm-devices/D1-bundle/select-your-station.png")} />
+        <img style={{width:"calc(100%*1/3 - 10px)"}} src={useBaseUrl("img/wxm-devices/D1-bundle/D1-claim.jpeg")} />
```

Comment:
The image is wrong here. Please use an image from the app showing the "Scan QR" step.